### PR TITLE
alexander_hristov/io: console reader and reverse file task

### PR DESCRIPTION
**Pull request by aleksandrhristov**
Third task from IO

### CODE REVIEW

File: input-output/src/reversecontent/Demo1.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package reversecontent;
+import java.io.IOException;
+/**
+ * Created by clouway on 10.05.16.
+ */
+public class Demo1 {
+    public static void main(String[] args) throws IOException {
+        java.io.File fileToRead = new java.io.File("files.txt");
+        Text demo = new Text();
```
Comment (mgenov) [inline input-output/src/reversecontent/Demo1.java]: Text text = new Text();
text.reverse(databaseFile);

**Comment by mgenov** [comment]
LGTM !

**Comment by aleksandrhristov** [comment]
new task ready for review

File: input-output/src/reversecontent/Text.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package reversecontent;
+
+import java.io.*;
+
+/**
+ * Created by clouway on 10.05.16.
+ */
+public class Text {
+
```
Comment (mgenov) [inline input-output/src/reversecontent/Text.java]: empty lines could be removed.

**Comment by mgenov** [comment]
As I can see PR is accepted. Why it's not merged ?

**Comment by aleksandrhristov** [comment]
I though i need to complete all the tasks to merge

**Comment by mgenov** [comment]
No, each task should be on it's own PR.

**Comment by aleksandrhristov** [comment]
ok

**Comment by mgenov** [comment]
???

**Pull request closed without merging**